src/blog/stylo-another-kind-of-rich-text-editor.md

```diff
@@ -0,0 +1,164 @@
+
+This decision was made for performance reason. In the future we might need to lazy load the content or improve the scrolling experience with some sort of infinite or other virtual scrolling approach. By limiting children to elements, it is possible to assign attributes such as unique ids directly in the DOM. <br>
+
+This brings us to the customization. The list of paragraphs that can be added by the user is fully customizable. One entry — i.e. a plugin — in the list is a text, an icon and a function that adds a new element to the editable container.
+
+For example, if we would implement a plugin that generates a new paragraph that is itself a Web Component name <hello-world/>, the solution would look as following:
+
+```javascript
+import {createEmptyElement, StyloPlugin, StyloPluginCreateParagraphsParams, transformParagraph} from '@papyrs/stylo';
+export const hr: StyloPlugin = {
+  text: 'My Hello World',
+  icon: `<svg width="32" height="32" viewBox="0 0 512 512">
+        ...
+    </svg>
+  `,
+  createParagraphs: async ({container, paragraph}: StyloPluginCreateParagraphsParams) => {
+    // Create your Web Component or HTML Element
+    const helloWorld = document.createElement('hello-world');
+    // Set properties, attributes or styles
+    helloWorld.setAttributes('yolo', 'true');
+     transformParagraph({
+       elements: [helloWorld, createEmptyElement({nodeName: 'div'})],
+       paragraph,
+       container,
+       focus: 'first'
+     })
+  }
+};
+```
+
+Stylo exposes interfaces and utilities to ease the development of custom plugins. The most important being maybe transformParagraph which parses the elements to the DOM.
+
+Finally, it is worth to note the function createParagraphs is async which means the process of creating new elements can totally be hijacked by your application. As for example, when integrated in DeckDeckGo, the function does not always directly create new elements. For some more complex use cases, when the function is triggered, it opens modals that require more user interactions such as selecting an image before effectively modifying the content.
+
+![](https://cdn-images-1.medium.com/max/1600/1*9GgkGbx_4D-hIwT8PxJivQ.gif)
+
+*****
+
+#### Framework Agnostic 😎
+
+Stylo is a framework agnostic library of reusable components developed with [StencilJS](https://stenciljs.com/). It works on all modern browsers.
+
+*****
+
+#### Lightweight 🪶
+
+30kb gzipped might in certain circumstances already sound like a lot, specially to me, but in terms of WYSIWYG editor it actually turns out be lightweight.
+
+From what I observed, it might even be the lightest. There is also probably a bit of space for improvements.
+
+*****
+
+#### Future Proof 🚀
+
+While a handful of actions in the alpha version still rely on execCommand to apply styles (e.g. bold, italic) — the core of the library does not.
```

Comment:
```suggestion
While a handful of actions in the alpha version still rely on `execCommand` to apply styles (e.g. bold, italic) — the core of the library does not.
```

Been adding those backticks, but actually maybe you don't `mark` the words on purpose? From now on I'll refrain from adding backticks